<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Maniteja9101996" failures="2" tests="2" name="Test(failed)(failed)" time="0.003" errors="0" timestamp="2023-10-06T11:58:46 IST">
  <testcase classname="runFailesTC.ClassA" name="Testcase2" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at runFailesTC.ClassA.Testcase2(ClassA.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at runFailesTC.FailTestRunnner.RunFailTestcases(FailTestRunnner.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 55 stack frames]]>
    </failure>
  </testcase> <!-- Testcase2 -->
  <testcase classname="runFailesTC.ClassA" name="Testcase1" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at runFailesTC.ClassA.Testcase1(ClassA.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at runFailesTC.FailTestRunnner.RunFailTestcases(FailTestRunnner.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 55 stack frames]]>
    </failure>
  </testcase> <!-- Testcase1 -->
</testsuite> <!-- Test(failed)(failed) -->
